packages/core/src/tracing/idleSpan.ts

```diff
@@ -130,7 +130,7 @@ export function startIdleSpan(startSpanOptions: StartSpanOptions, options: Parti
       // If we have no spans, we just end, nothing else to do here
       if (!spans.length) {
         onIdleSpanEnded(spanEndTimestamp);
-        return Reflect.apply(target, thisArg, args);
```

Comment:
Is this change intended?

Reply:
Hard to do with a type as stuff would always have to be optional, so this would always match - but I added code to ensure we also pass rest arguments, so that is forwards-compatible!